src/main/java/net/neoforged/neoforge/client/ClientNeoForgeMod.java

```diff
@@ -24,11 +27,13 @@
 
 @Mod(value = "neoforge", dist = Dist.CLIENT)
 public class ClientNeoForgeMod {
```

Comment:
Could you slip in an internal annotation for this class? It's technically an internal class that isn't for modders to call

Reply:
sure can do